daprdocs/content/en/go-sdk-docs/go-client/_index.md

```diff
@@ -187,7 +183,6 @@ out, err := client.InvokeBinding(ctx, in)
 
 For a full guide on output bindings, visit [How-To: Use bindings]({{< ref howto-bindings.md >}}).
 
-### Actors
```

Comment:
Same as above.

Reply:
@daixiang0 Until we have content for that section, I'd rather remove completely from the docs - unless you have a go-sdk example for actors that I can use here?